### DISCUSSION TITLE
RC output of UART on Radiomaster TX16s

### DISCUSSION THREAD

**Discussion by benjinne** [EdgeTX/edgetx#1908] [Q&A]
Hi, is it possible to configure a UART on the Radiomaster TX16s to output RC like the JR module or internal module?
For context, I'm trying to use 3.3V logic levels and output CRSF to an external transmitter that simply forwards a UART over IP and there's no JR module for this transmitter. My test case to get this working is to directly connect a transmitter to a flight controller and see if it talks without any wireless communication involved. I tried using the JR module pins and outputting CRSF, but it doesn't seem to talk directly to a flight controller. I did however get sbus working.

**Comment by aim-to-be-free** [comment]
@benjinne hey, i am interested in that too!
Have you learned it?
